### PR TITLE
New oru component

### PR DESCRIPTION
**Pull request by bvlaicu**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27571

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/oru.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Orange and Rockland Utility Real Time Energy Usage Sensor"
+description: "Instructions on how to integrate the Orange and Rockland Utility real time energy usage sensor within Home Assistant."
+logo: oru.png
+ha_release: 0.???
```
Comment (klaasnicolaas) [inline source/_integrations/oru.markdown]:
```suggestion
ha_release: 0.101
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

Happy Hacktoberfest! 🎉

File: source/_integrations/oru.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Orange and Rockland Utility Real Time Energy Usage Sensor"
+description: "Instructions on how to integrate the Orange and Rockland Utility real time energy usage sensor within Home Assistant."
+logo: oru.png
```
Comment (bvlaicu) [inline source/_integrations/oru.markdown:4]: @klaasnicolaas Any idea where the logo image file should be hosted? Thanks.

Comment (frenck) [reply source/_integrations/oru.markdown:4]: @bvlaicu It should be in the folder: `source/images/supported_brands`

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request merged**